### ISSUE TITLE
Doesn't work when run from a Linux agent

### ISSUE THREAD

**Issue by danielthompson** [BMuuN/vsts-assemblyinfo-task#50]
When running this task in a VSTS pipeline on a Linux agent, I get the following error:

```
2018-08-27T02:59:37.9652625Z ##[section]Starting: Set Assembly Manifest Data
2018-08-27T02:59:37.9767946Z ==============================================================================
2018-08-27T02:59:37.9783130Z Task         : Assembly Info
2018-08-27T02:59:37.9798682Z Description  : Set assembly information such as Version, Copyright and Trademark etc.
2018-08-27T02:59:37.9813597Z Version      : 1.1.39
2018-08-27T02:59:37.9828624Z Author       : Bleddyn Richards
2018-08-27T02:59:37.9845299Z Help         : [More Information](https://marketplace.visualstudio.com/items?itemName=bleddynrichards.Assembly-Info-Task)
2018-08-27T02:59:37.9862464Z ==============================================================================
2018-08-27T02:59:38.0827851Z ##[error]The current operating system is not capable of running this task. That typically means the task was written for Windows only. For example, written for Windows Desktop PowerShell.
2018-08-27T02:59:38.0881242Z ##[section]Finishing: Set Assembly Manifest Data
```

**Comment by SeriousM** [comment]
#52 has more information about this issue

**Comment by BMuuN** [comment]
This issue has been fixed with v2.0.65. Apologies for the delay implementing this change!

Please note this upgrade is a breaking change and you should follow these steps to prevent any down-time:  https://github.com/BMuuN/vsts-assemblyinfo-task/wiki/Upgrade-v1-to-v2

This issue will now be closed.  If you feel the issue has not been fixed please feel free to re-open the issue.
